tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -133,14 +133,44 @@ test.describe( 'Set up accounts', () => {
 			await setUpAccountsPage.goto();
 		} );
 
-		test( 'should see their WPORG email, "Google" title & connect button', async () => {
-			const jetpackDescriptionRow =
-				setUpAccountsPage.getJetpackDescriptionRow();
+		test( 'should not show the WP.org connection card when already connected', async () => {
+			await expect(
+				page.getByRole( 'heading', { name: 'Set up your accounts' } )
+			).toBeVisible();
+
+			await expect(
+				page.getByText(
+					'Connect the accounts required to use Google for WooCommerce.'
+				)
+			).toBeVisible();
+
+			await expect(
+				page.getByRole( 'button', { name: 'Connect' } ).first()
+			).toBeEnabled();
```

Comment:
It would be a bit clearer to assert this button by specifying the Google account card.

https://github.com/woocommerce/google-listings-and-ads/blob/9e606c1b80d164f770ed7d5c29e8272a6ae0d62f/tests/e2e/specs/setup-mc/step-1-accounts.test.js#L180-L184

The `.first()` in the code reference could be omitted.

Reply:
This assertion and the two above it seem to be copied from:

https://github.com/woocommerce/google-listings-and-ads/blob/9e606c1b80d164f770ed7d5c29e8272a6ae0d62f/tests/e2e/specs/setup-mc/step-1-accounts.test.js#L51-L63

Assuming the original intention of this assertion placed here is not to check the connect button in *Google account card*, considering the same assertion will be performed by another test case, then I would suggest removing this assertion.